apps/devhub/tests/test_utils.py

```diff
@@ -0,0 +1,373 @@
+import os.path
+from copy import deepcopy
+
+import mock
+import pytest
+from mock import call
+
+from django.conf import settings
+
+import amo
+import amo.tests
+from addons.models import Addon
+from files.models import File, FileUpload
+from versions.models import Version
+from devhub import utils
+from devhub.tasks import annotate_validation_results
+
+
+pytestmark = pytest.mark.django_db
+
+
+def change_dict(src, changes):
```

Comment:
Could that be named something like `new_updated_dict`? You're calling it `change_dict` while you're explicitly not mutating it.